src/UI/citizens/components/header/CitizensHeaderStyle.js

```diff
@@ -1,12 +1,353 @@
 import styled from 'styled-components';
 
 const HeaderContainer = styled.header`
-  width: 100%;
+  width: 1440px;
   padding: 20px;
 `;
 
 const HeaderTitle = styled.h1`
-  color: red;
+  width: 576px;
+  height: 40px;
+  font-family: Lora;
+  font-size: 32px;
+  font-weight: bold;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #000000;
+`;
+
+const HeaderButton = styled.div`
+  width: 108px;
+  height: 24px;
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0f4850;
+`;
+
+const HeaderImage = styled.img`
```

Comment:
O ovome smo već pričali pa znaš da to treba riješiti sa css-om, ne sa img elementom